spec/design/skel/Structs_B.thy

```diff
@@ -0,0 +1,24 @@
+(*
+ * Copyright 2024, Proofcraft Pty Ltd
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
+(* Generic data types shared by spec and abstract. *)
+
+chapter "Common, Generic Data Types"
```

Comment:
I don't want to nitpick too much, but reading these now...

- "Architecture-specific data types shared by spec and abstract."
- "Generic data types shared by spec and abstract."

Um, between spec and abstract? That makes no sense. "design and abstract specs"?

The "Common" in the chapter titles also takes a moment to think through, and only clicked when I read the comments above... but it probably makes sense to keep that.

Reply:
Agreed that your suggestion makes more sense. I didn't think about it much before and just mindlessly copied the existing comment from `Arch_Structs_B.thy`